### DISCUSSION TITLE
Mutiple cucumber html report - Aggregated report (WDIO v7)

### DISCUSSION THREAD

**Discussion by gopilaxman** [WasiqB/multiple-cucumber-html-reporter#237] [Q&A]
Mutiple HTML cucumber report in my repo. When I try to rerun the failed test case, It always adds new entry in the HTML report which shows duplicate entries in the report. And test case counts are increased including failed testcases + Passed Testcases.
It makes little messy after couple of runs. Is there a way to updated existing rows instead of adding new entries for every re-run ?
Is there a way to aggregate the report like the behavior in allure? Thanks in advance..

**Comment by WasiqB** [comment]
Hi @gopilaxman ,

Thanks for starting a discussion for the issue you are facing.

Could you help us with a sample repo where you are using this report? We will look into it and see how we can improve on this behaviour.

**Comment by gopilaxman** [comment]
![image](https://user-images.githubusercontent.com/42570287/220353680-ee3e880f-2340-49ba-930f-ae17d2c41eb3.png)
